code/edgeidentity/build.gradle

```diff
@@ -31,6 +33,7 @@ android {
 
         testInstrumentationRunner rootProject.ext.testInstrumentationRunner
         consumerProguardFiles "consumer-rules.pro"
+        multiDexEnabled true
```

Comment:
Need to add
namespace 'com.adobe.marketing.edge.identity.app'
to line 28.
It was removed from AndroidManifest.xml.

Reply:
This gradle file corresponds to the library. `namespace` for the testapp has been declared in its .gradle

Reply:
We have another AndroidManifest.xml at 
https://github.com/prudrabhat/aepsdk-edgeidentity-android/blob/dev-v2.0.0_setup/code/edgeidentity/src/main/AndroidManifest.xml